center_loss/face_model.py

```diff
@@ -0,0 +1,61 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.multiprocessing as mp
+import torch.utils.model_zoo as model_zoo
+from torchvision.models.resnet import BasicBlock, ResNet, model_urls
+import math
+
+class FaceModel(nn.Module):
+    def __init__(self,num_classes, pretrained=False, **kwargs):
+        super(FaceModel, self).__init__()
+        self.model = ResNet(BasicBlock, [2, 2, 2, 2], **kwargs)
```

Comment:
This is resnet18, but in the `pretrained` condition you load `resnet50`. Could you fix it?

Reply:
How do I divide LFW dataset into test/train? Some class only have 1 image.

Reply:
The author put it only in training set according to the 2 lines:
```
val_indices.extend(indexes[:int(np.floor(len(indexes)*split))])
train_indices.extend(indexes[int(np.floor(len(indexes)*split)):])
```
In my code, I put it both on validation and training set. I'm also not sure if it's ok to do that. You can reach the code at https://github.com/louis-she/center-loss.pytorch/blob/master/dataset.py#L65

Reply:
Just found out that one should really use `pairsDevTest.txt` to get the evaluation pairs. Suggested by FLW official. Refer to http://vis-www.cs.umass.edu/lfw/README.txt